samples/rag-cosmosdb-nosql/python/function_app.py

```diff
@@ -0,0 +1,37 @@
+import json
```

Comment:
please add a file - local.settings.json with required settings and placeholder values like existing samples

Reply:
The files were there, but it got added to .gitignore, so it didn't get comitted.